### DISCUSSION TITLE
Contact support?

### DISCUSSION THREAD

**Discussion by adams-family** [taligentx/dscKeybusInterface#274] [dscKeybusInterface Discussion]
Hi,

Excellent work!

Is it possible to contact the author of this library for support purposes? I will pay / donate of course to compensate for the spent time.

Thank you!

**Comment by taligentx** [comment]
Hi @adams-family - could you detail what you're looking for assistance with?  There are several helpful folks that have assisted other users here in the issues/discussions - if you're running into issues getting things going, it's likely other folks may also run into the same problems and so it'd benefit all to have that documented here in the discussions for future reference.  If it looks like something that would be best done one-to-one we can look into more dedicated support options, thanks!

-Nikhil
